### ISSUE TITLE
A bit more containment

### ISSUE THREAD

**Issue by righteousbs** [jimz011/homekit-infused#49]
It would be good to keep following HKI code blocks contained within the package for those of us that use multiple dashboards. Suggest placing these blocks within ...packages/homekit_infused/hki_configuration.yaml

```
sensor:
  - platform: time_date
    display_options:
        ....
```
```
lovelace:
  dashboards:
    homekit-infused:
      mode: yaml
      title: Homekit Infused
      icon: mdi:home-assistant
      show_in_sidebar: true
      filename: homekit-infused-lovelace.yaml
```

**Comment by jimz011** [comment]
I'm not entirely understanding what you mean with multiple dashboards (since the sensor you mention is just a sensor and wouldn't affect other dashboards). However the idea of putting the sensor there is a good one and I will do that!

(please elaborate on the multi dashboard comment) I'll merge this anywayws!

**Comment by jimz011** [comment]
Ah I get it, you mean the dashboard part? Yeah I could put that in the package as well, wouldnt really matter since you could just add other dashboards to the same file. I am not sure how a package will handle multiple dashboards, but I will test this tonight. Good thinking though! Appreciate it

**Comment by righteousbs** [comment]
Yeah to both, thought it better suited to be within the package rather than outside of it.  The sensor, at least in my case, is only applicable to the HKI package, and the dashboard is specific to HKI.  You can define the lovelace: dashboard stuff per package as long as the mode definition doesn't conflict between packages.

**Comment by jimz011** [comment]
Alright I'll push these changes with the next update.

**Comment by jimz011** [comment]
@righteousbs I might as well put the manual_alarm in there as well no?

**Comment by righteousbs** [comment]
Makes sense, it's a required feature for HKI.

**Comment by jimz011** [comment]
Gonna close this issue since it has been fixed for v3.3.0 (release soon)
